sdk/nodejs/runtime/settings.ts

```diff
@@ -64,7 +84,7 @@ export function _setIsDryRun(val: boolean) {
  * and therefore certain output properties will never be resolved.
  */
 export function isDryRun(): boolean {
-    return options.dryRun === true || isTestModeEnabled();
+    return options.dryRun === true;
```

Comment:
Will this change break folks who are using the old test mode?

Reply:
It will, yes--do you have a suggestion for avoiding this?